packages/upgrade/test/fixtures/upgrade/ts-errors.ts.output

```diff
@@ -35,32 +22,13 @@ export function usesMissingVarWithLongName(): boolean {
       missingVarAsWellButThisTimeWeUseVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongString;
 }
 
-/* @ts-ignore @rehearsal TODO TS6133: The function 'notProperlyFormatted' is never called. Remove the function or use it. */
-function notProperlyFormatted(file: string): string {
-  /* @ts-ignore @rehearsal TODO TS2304: Cannot find name 'doSomething'. */
-  return doSomething(file);
-}
-
-/**
- * Existing multiline comment
- */
-/* @ts-ignore @rehearsal TODO TS6133: The function 'thisFunctionIsACompleteMess' is never called. Remove the function or use it. */
-function thisFunctionIsACompleteMess(inSeconds = false): string {
-  // Existing comment
-  const resolved = resolve('a');
-  /* @ts-ignore @rehearsal TODO TS2304: Cannot find name 'foo'. */
-  foo = '6';
-  /* @ts-ignore @rehearsal TODO TS6133: The function 'timestamp' is never called. Remove the function or use it. */
-  function timestamp(inMinutes = false): void {
-    /* @ts-ignore @rehearsal TODO TS6133: The variable 'b' is never read or used. Remove the variable or use it. */
-    const b = null;
-    /* @ts-ignore @rehearsal TODO TS2322: The function expects to return 'void', but 'string' is returned. Please convert 'string' value to 'void' or update the function's return type. */
-    return 'a';
-  }
+// The next function call is not properly formatted
+timestamp();
 
-  return resolved;
+function missingFunction(): void {
+  throw new Error('Function not implemented.');
```

Comment:
This is one of TS CodeFixes behavior... we can decide override this behavior by removing function (or add comment as before)